### ISSUE TITLE
InstallAssistant fails to install

### ISSUE THREAD

**Issue by cciRRus** [corpnewt/gibMacOS#296]
I've successfully downloaded the files with the script. The files are found in this folder: "052-69919 - 14.6 macOS Sonoma (23G80)"
But InstallAssistant.pkg fails complete:

![Screenshot 2024-08-03 at 9 57 35 AM](https://github.com/user-attachments/assets/9fc821fc-b9ea-4032-bcf6-b6718a9d7206)

Any ideas why?

**Comment by david-w** [comment]
I noticed the same thing, but it actually works.  It just reports as failed.  
The macOS installer is created in the Applications directory and installs successfully.

**Comment by Gravarty** [comment]
> I noticed the same thing, but it actually works. It just reports as failed. The macOS installer is created in the Applications directory and installs successfully.

No, it does not. The installer is just 58MB in size after it's installed into the Applications folder.

**Comment by prince1604** [comment]
okay
